index.rst

```diff
@@ -1,61 +1,383 @@
+- Logging and monitoring
 
-     Caption text.
+.. _SQR-042: https://sqr-042.lsst.io/
 
-   Run: ``make html`` and ``open _build/html/index.html`` to preview your work.
-   See the README at https://github.com/lsst-sqre/lsst-technote-bootstrap or
-   this repo's README for more info.
+However, the narrow topic of protecting web access to applications within the Rubin Science Platform from a compromise of a single Science Platform application is in scope insofar as it can be addressed through good credential management practices and effective use of separate JavaScript origins.
 
-   Feel free to delete this instructional comment.
+The Rubin Science Platform is different than most web applications in that it intentionally offers its users remote code execution as a service via the Notebook Aspect.
+Authenticated users are therefore expected to have considerable latitude in what actions they can perform.
+The focus of this discussion is therefore on:
 
-:tocdepth: 1
+- Preventing unauthenticated access
+- Preventing credential theft or hijacking (such as via CSRF)
+- Protecting Rubin Science Platform users from malicious activity by other users
+- Isolating services from each other to provide some defense in depth against security vulnerabilities in Science Platform services
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+Glossary
+--------
 
-.. sectnum::
+Cross-site request forgery (CSRF)
+    An attack that tricks the user into sending a malicious request on the attacker's behalf.
+    The most obvious mechanism to do this is cross-site ``GET`` or ``POST`` requests triggered by JavaScript run on a malicious site to which the user went in a web browser, although there are other mechanisms.
+    CSRF generally exploits the browser's security model to trick the browser into sending credentials that it has stored for its user (such as cookies) along with a request that is controlled by the attacker.
+
+Cross-site scripting (XSS)
+    An attack that involves injecting malicious content (usually JavaScript or CSS, although there are other possibilities) into a trusted web site, such as through untrusted user input that is displayed unescaped to other users.
+    The malicious code then runs within the security context of the trusted site and can access the user's stored authentication credentials for that site, allowing it to take actions on behalf of the user, steal the credentials, or take other malicious actions.
+
+Threat model
+============
+
+The expected goals of an attacker targeting the Science Platform are primarily the standard goals for general Internet attackers:
+
+- Theft of compute resources (Bitcoin mining, bot networks)
+- Extortion via ransomware (CryptoLocker)
+- Web site hosting for further phishing or malware distribution
+- Exfiltration of confidential data (of which there is very little on the Rubin Science Platform)
```

Comment:
I know what you mean and that  it is the correct use of the term, but you might want to add "personal" or some other modifier to "confidential data" to make sure the reader doesn't just think of prorprietary user data here. 
(I know this is also addressed at length before but people who skim read are drawn to bullet points)